s19/examples.rb

```diff
@@ -0,0 +1,45 @@
+# example 1
+module LengthConversions
+  WEBSITE = "https://google.com"
+
+  def self.meters_to_kilometers(m)
```

Comment:
What's the difference between these two methods?

```
  def self.meters_to_kilometers(m)
    m / 1000.0
  end
```

```
  def meters_to_kilometers(m)
    m / 1000.0
  end
```

Reply:
That the word Self allows methods to belong to the module and not to an instance of it, something that by the definition of module is not possible.